make/conf/jib-profiles.js

```diff
@@ -999,25 +999,36 @@ var getJibProfilesProfiles = function (input, common, data) {
         testOnlyProfilesPrebuilt["run-test-prebuilt"]["dependencies"].push(testedProfile + ".jdk_symbols");
     }
 
+    var testOnlyProfilesPrebuiltDocs = {
+        "run-test-prebuilt-docs": clone(testOnlyProfilesPrebuilt["run-test-prebuilt"])
+    };
+
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].dependencies.push("docs.doc_api_spec", "tidy");
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["DOCS_IMAGE_DIR"] = input.get("docs.doc_api_spec", "install_path");
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["TIDY_HOME"] = input.get("tidy", "home_path") + "/bin/tidy";
```

Comment:
This should be all you need for supplying `TIDY` to the makefiles when running tests without configure.
```suggestion
    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["TIDY"] = input.get("tidy", "home_path") + "/bin/tidy";
```